src/plugin/context.rs

```diff
@@ -880,4 +883,28 @@ impl RapierContext {
             )
         });
     }
+
+    /// Computes the angle between the 2 bodies
+    /// referenced by the underlying [`ImpulseJoint`] of the given `entity`,
+    /// according to its [`RevoluteJoint`] description.
+    ///
+    /// Parameter `entity` should have a [`ImpulseJoint`] or [`MultibodyJoint`] component with a [`JointDescription::RevoluteJoint`] variant as `data`.
```

Comment:
Doesn’t look like the code handle multibody joints:
```suggestion
    /// Parameter `entity` should have a [`ImpulseJoint`] component with a [`JointDescription::RevoluteJoint`] variant as `data`.
```